src/Stack.cpp

```diff
@@ -0,0 +1,153 @@
+#pragma once
+#include "Stack.h"
+
+
+Stack::Stack(const size_t& size, const int& value)
+	:Vector::Vector(size, value)
+{
+}
+
+// Copy constructor
+Stack::Stack(const Stack& other) {
+	*this = other;//use Vector Copy Assiment
+}
+
+// Move Constructor
+Stack::Stack(Stack&& other) noexcept
+{
+	Vector::operator=(other);
```

Comment:
If you use the exact same method of the parent, why even implement the method? 
If C++'s polymorphism mechanism doesn't see an implementation it will just default to use the parent's method